pages/docs/input-data/crac/csa.md

```diff
@@ -0,0 +1,758 @@
+        <cim:IdentifiedObject.mRID>contingency-equipment</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.name>Contingency equipment</cim:IdentifiedObject.name>
+        <cim:IdentifiedObject.description>Example of contingency equipment.</cim:IdentifiedObject.description>
+        <cim:ContingencyElement.Contingency rdf:resource="#_out-of-range-contingency"/>
+        <cim:ContingencyEquipment.contingentStatus
+                rdf:resource="http://iec.ch/TC57/CIM100#ContingencyEquipmentStatusKind.outOfService"/>
+        <cim:ContingencyEquipment.Equipment rdf:resource="#_equipment"/>
+    </cim:ContingencyEquipment>
+    ...
+</rdf:RDF>
+```
+
+{% endcapture %}
+{% include /tabs.html id="CSA_CO_tabs" tab1name="ExceptionalContingency" tab1content=case_ExceptionalContingency
+tab2name="OutOfRangeContingency" tab2content=case_OutOfRangeContingency %}
+
+A contingency is imported only if the `normalMustStudy` field is set to `true` and if it is referenced by a
+valid `ContingencyEquipment`, i.e. having `Equipment` pointing to an existing network element and a `contingentStatus`
+being `outOfService`.
+
+From the `ExceptionalContingency` / `OutOfRangeContingency` object, the `mRID` is used as the contingency's identifier.
+Besides, the `EquipmentOperator` is converted to a friendly name and concatenated with the `name` to create the
+contingency's name (if the TSO is missing, only the name is used; if the name is missing, the `mRID` will be used
+instead). Finally, the `ContingencyEquipment`'s `Equipment` is used as the contingency's network element.
+
+## CNECs {#cnecs}
+
+The [CNECs](json#cnec) are described in the **AE** profile with an `AssessedElement` object which bears the identifier,
+name, instant(s) and operator information.
+
+```xml
+<!-- AE Profile -->
+<rdf:RDF>
+    ...
+    <nc:AssessedElement rdf:ID="_assessed-element">
+        <cim:IdentifiedObject.mRID>assessed-element</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.name>Assessed element</cim:IdentifiedObject.name>
+        <cim:IdentifiedObject.description>Example of assessed element.</cim:IdentifiedObject.description>
+        <nc:AssessedElement.inBaseCase>true</nc:AssessedElement.inBaseCase>
+        <nc:AssessedElement.isCritical>true</nc:AssessedElement.isCritical>
+        <nc:AssessedElement.normalEnabled>true</nc:AssessedElement.normalEnabled>
+        <nc:AssessedElement.isCombinableWithRemedialAction>false</nc:AssessedElement.isCombinableWithRemedialAction>
+        <nc:AssessedElement.isCombinableWithContingency>false</nc:AssessedElement.isCombinableWithContingency>
+        <nc:AssessedElement.AssessedSystemOperator rdf:resource="http://data.europa.eu/energy/EIC/10XFR-RTE------Q"/>
+        <nc:AssessedElement.OperationalLimit rdf:resource="#_operational-limit"/>
+    </nc:AssessedElement>
+    ...
+</rdf:RDF>
+```
+
+The CNEC is imported only if the `isCritical` and `normalEnabled` fields are both set to `true`. If the `inBaseCase`
```

Comment:
The CNEC is also imported if these fields are missing, because some TSO won't use them.